packages/hapi/modules/plugin/plugin.js

```diff
@@ -81,7 +83,16 @@ const createPlugin = ({ options } = {}) => {
       server.decorate('server', 'Prometheus', Prometheus);
       server.decorate('server', 'recordRequest', recordRequest);
 
-      return onDone && onDone();
+      server.route({
```

Comment:
This should be done here. You'd use `@promster/server` or wire up the route yourself in your app. I recommend the first as then the Prometheus server runs on another port. That eases not exposes metrics e.g. outside a k8s cluster.

Reply:
as I commented earlier, I think provide a default route and configurable will be more flexible and developer friendly.